WcaOnRails/spec/spec_helper.rb

```diff
@@ -84,4 +84,15 @@
   # test failures related to randomization by passing the same `--seed` value
   # as the one that triggered the failure.
   Kernel.srand config.seed
+
+  config.before(:suite) do
+    %w(Countries Continents Events Rounds Formats teams).each do |table|
+      # -- Uncomment to regenerate --
+      # ActiveRecord::Base.connection.execute("TRUNCATE #{table};")
+
+      if ActiveRecord::Base.connection.execute("SELECT COUNT(*) FROM #{table}").first == [0]
+        load "#{Rails.root}/db/seeds/#{table.underscore}.seeds.rb"
```

Comment:
@jonatanklosko @larspetrus With the Countries, Continents, Events, Rounds, and Formats this isn't actually a problem. Because they are _virtual_ a migration would need to be written to keep the production database up to date. With the `teams` table though, this could be created through the front end so it is an issue.

We may be over complicating this. it might be simpler to truncate and reload these tables each time. At the moment we are doing one select per table every time to check if they are empty. If we also add the checksum logic it will take a little bit more time. These tables are all small (Continents is the largest with 199 rows, and most are less than 10 rows) so loading them is very quick. This means it won't be much slower to truncate and load each time. As a side note I think we currently insert the 4 teams records before each test so this will be faster than what currently occurs.

I suggest that for this PR I simply truncate and load these tables once each time rspec is called. We can then raise a separate issue if we think this needs to be improved by adding a checksum and only reloading the tables if a seed file has changed. Let me know your thoughts.